docs/storage-abstraction.md

```diff
@@ -47,157 +69,58 @@ You can create a JSON file emoney-storage.json with the JSON below.
 
-  private final DistributedStorage storage;
+{% capture notice--warning %}
+**Attention**
 
-  public ElectronicMoney() throws IOException {
-    StorageFactory factory = StorageFactory.create(SCALARDB_PROPERTIES);
-    storage = factory.getStorage();
-  }
+As previously mentioned, since the Storage API cannot provide transaction capability, the API could cause anomalies or data inconsistency if failures occur when executing operations. Therefore, you should be *very* careful about using the Storage API and use it only if you know exactly what you are doing.
+{% endcapture %}
 
-  public void charge(String id, int amount) throws ExecutionException {
-    // Retrieve the current balance for id
-    Get get =
-        Get.newBuilder()
-            .namespace(NAMESPACE)
-            .table(TABLENAME)
-            .partitionKey(Key.ofText(ID, id))
-            .build();
-    Optional<Result> result = storage.get(get);
-
-    // Calculate the balance
-    int balance = amount;
-    if (result.isPresent()) {
-      int current = result.get().getInt(BALANCE);
-      balance += current;
-    }
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
 
-    // Update the balance
-    Put put =
-        Put.newBuilder()
-            .namespace(NAMESPACE)
-            .table(TABLENAME)
-            .partitionKey(Key.ofText(ID, id))
-            .intValue(BALANCE, balance)
-            .build();
-    storage.put(put);
-  }
+### Reference
 
-  public void pay(String fromId, String toId, int amount) throws ExecutionException {
-    // Retrieve the current balances for ids
-    Get fromGet =
-        Get.newBuilder()
-            .namespace(NAMESPACE)
-            .table(TABLENAME)
-            .partitionKey(Key.ofText(ID, fromId))
-            .build();
-    Get toGet =
-        Get.newBuilder()
-            .namespace(NAMESPACE)
-            .table(TABLENAME)
-            .partitionKey(Key.ofText(ID, toId))
-            .build();
-    Optional<Result> fromResult = storage.get(fromGet);
-    Optional<Result> toResult = storage.get(toGet);
-
-    // Calculate the balances (it assumes that both accounts exist)
-    int newFromBalance = fromResult.get().getInt(BALANCE) - amount;
-    int newToBalance = toResult.get().getInt(BALANCE) + amount;
-    if (newFromBalance < 0) {
-      throw new RuntimeException(fromId + " doesn't have enough balance.");
-    }
+To see the source code for the electronic money application used in this tutorial, see [`ElectronicMoney.java`](https://scalardb.scalar-labs.com/docs/latest/getting-started/src/main/java/sample/ElectronicMoney.java).
```

Comment:
This code isn't for the Storage API, but the Transaction API. I don't think we have a separate source code file for this. So, I think we need to restore the Java code.

Reply:
Understood. I've re-added the example code here.